osc-server/src/main/java/org/osc/core/broker/service/tasks/conformance/k8s/securitygroup/UpdateK8sSecurityGroupMemberLabelMetaTask.java

```diff
@@ -89,7 +89,9 @@ public void executeTransaction(EntityManager em) throws Exception {
         Set<Pod> dbPodsToDelete = emptyIfNull(label.getPods()).stream()
                 .filter(p -> !existingPodIdsInK8s.contains(p.getExternalId())).collect(Collectors.toSet());
 
-        k8sPodsToCreate.forEach(p -> this.tg.addTask(this.labelPodCreateTask.create(p, this.sgm.getLabel())));
+        // TODO emanoel: Using append instead of adding here because of a transactional issue updating multiple
+        // pods in the same label concurrently. An alternative to this might be locking the label member entity.
+        k8sPodsToCreate.forEach(p -> this.tg.appendTask(this.labelPodCreateTask.create(p, this.sgm.getLabel())));
         dbPodsToDelete.forEach(p -> this.tg.addTask(this.labelPodDeleteTask.create(p, this.sgm.getLabel())));
```

Comment:
would this impact delete the same way? should this be append as well?

Reply:
The deletion is not updating the label entity, just deleting the pod and port so i dont think it will have the same impact, I will cover that by deleting two pods on the same label and syncing to make sure. If there is an issue I will update the PR with the same workaround before merging.